### PR TITLE
pgtype Version 1.8.0 test and validation script

### PR DESCRIPTION
**Pull request by ambujkumar3**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@prashantkhoje please review

File: p/pgtype/pgtype_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+#cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! (go build); then
+       echo "------------------$PACKAGE_NAME:build failed---------------------"
+       echo "$PACKAGE_VERSION $PACKAGE_NAME"
+       echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  build_Fails"
+       exit 1
+fi
+echo $PWD
```
Comment (prashantkhoje) [inline p/pgtype/pgtype_ubi_8.5.sh:124]:
```suggestion
# Check if package exists
if [ -d "$PACKAGE_NAME" ] ; then
    rm -rf $PACKAGE_NAME
    echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
fi

if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
    echo "$PACKAGE_URL $PACKAGE_NAME"
    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
    exit 0
fi
cd /home/tester/$PACKAGE_NAME
git checkout $PACKAGE_VERSION

if ! (go build); then
    echo "------------------$PACKAGE_NAME:build failed---------------------"
    echo "$PACKAGE_VERSION $PACKAGE_NAME"
    echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  build_Fails"
    exit 1
fi
echo $PWD
```

File: p/pgtype/pgtype_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" > /home/tester/output/test_fails
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails" > /home/tester/output/version_tracker
+        exit 1
+else
+        echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
+        echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_success
+        echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > /home/tester/output/version_tracker
+        exit 0
+fi
```
Comment (prashantkhoje) [inline p/pgtype/pgtype_ubi_8.5.sh:147]:
```suggestion

if ! PGX_TEST_DATABASE="host=127.0.0.1 port=5433 database=pgx_test user=pgx_pw password=secret sslmode=disable" go test -v ./...; then
    echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
    echo "$PACKAGE_URL $PACKAGE_NAME" > /home/tester/output/test_fails
    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails" > /home/tester/output/version_tracker
    exit 1
else
    echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
    echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_success
    echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > /home/tester/output/version_tracker
    exit 0
fi
```

**Review by prashantkhoje — CHANGES_REQUESTED** [review]
Formatted script

**Comment by prashantkhoje** [comment]
LGTM

**Pull request merged**